warehouse/packaging/tasks.py

```diff
@@ -111,3 +113,54 @@ def update_description_html(request):
     for description in descriptions:
         description.html = readme.render(description.raw, description.content_type)
         description.rendered_by = renderer_version
+
+
+@tasks.task(
+    bind=True,
+    ignore_result=True,
+    acks_late=True,
+    autoretry_for=(Exception,),
+    retry_backoff=15,
+    retry_jitter=False,
+    max_retries=5,
+)
+def upload_bigquery_distributions(task, request, file, form):
```

Comment:
`update_bigquery_release_files` might be a better name to match things up across the DB models and service names.